ministryofjustice/peoplefinder
# CT-2520 add survey banner

## Description
Launching a survey on People Finder (PF), to understand current use, behaviour and the over-arching need for PF across the MoJ

We needed to add support for CircleCI 2.0 which now runs the tests on every commit

We also needed to update gem dependencies to address vulnerabilities reported by Github
 
## Self-review checklist

 
### Screenshots
![image](https://user-images.githubusercontent.com/1161161/67107654-c5725800-f1c4-11e9-97a2-5429bc8b4937.png)
![image](https://user-images.githubusercontent.com/1161161/67107683-d0c58380-f1c4-11e9-8c2b-b6f8b0615730.png)
![image](https://user-images.githubusercontent.com/1161161/67107701-da4eeb80-f1c4-11e9-8742-ec1f323edea8.png)

![image](https://user-images.githubusercontent.com/1161161/67105678-20a24b80-f1c1-11e9-8243-c2db45462357.png)

![image](https://user-images.githubusercontent.com/1161161/67105709-2f88fe00-f1c1-11e9-8244-096568606b87.png)
 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2572
https://dsdmoj.atlassian.net/browse/CT-2520

 
### Deployment
New gem versions, no changes to databases or versions of infrastructure OTHER than bumped Ruby to 2.3.7
 
### Manual testing instructions
Use the app - view the banner where it appears on home page, search results page, and Person profile page
